src/main/java/uk/gov/justice/digital/hmpps/keyworker/security/ResourceServerConfiguration.java

```diff
@@ -1,34 +1,35 @@
 package uk.gov.justice.digital.hmpps.keyworker.security;
 
+import java.util.Optional;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.data.domain.AuditorAware;
 import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
-import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.config.annotation.web.configurers.HeadersConfigurer.FrameOptionsConfig;
 import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.web.SecurityFilterChain;
 import org.springframework.stereotype.Service;
-import java.util.Optional;
 
 @Configuration
 @EnableJpaAuditing(auditorAwareRef = "auditorAware")
 @EnableWebSecurity
-public class ResourceServerConfiguration extends WebSecurityConfigurerAdapter {
-    
-    @Override
-    public void configure(final HttpSecurity http) throws Exception {
+public class ResourceServerConfiguration {
+    @Bean
+    public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
 
-        http.sessionManagement()
-            .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
-            .and().csrf().disable()// Can't have CSRF protection as requires session
-            .authorizeRequests(auth ->
-                auth.antMatchers("/webjars/**", "/favicon.ico", "/csrf",
-                        "/health/**", "/info", "/ping",
-                        "/v3/api-docs/**", "/v2/api-docs", "/swagger-ui/**", "/swagger-ui.html",
-                        "/swagger-resources", "/swagger-resources/configuration/ui", "/swagger-resources/configuration/security",
-                        "/queue-admin/retry-all-dlqs")
```

Comment:
Missed this from the paths we now pass to `httpRequest.requestMatchers`

Reply:
Done ✅